index.html

```diff
@@ -1,36 +1,51 @@
+        <link rel="stylesheet" href="main.css">
     </head>
-    <style>
-        table, thead{
-            background-color: wheat;
-            align-items: center;
-            text-align: center;
-        }
-    </style>
     <body>
-        <table border="1" style="border:3px blue solid;" class="table-responsive">
-            <thead>
-                <tr>
-                    <th>id</th>
-                    <th>Name</th>
-                    <th>UserName</th>
-                    <th>Email</th>
-                    <th>Address{street,suite,city,zipcode,geo(lat,lng)</th>
-                    <th>Phone</th>
-                    <th>Website</th>
-                    <th>company{name,catchPhrase,bs}</th>
-                </tr>
-            </thead>
-            <tbody id="data-output">
-
-            </tbody>
-        </table>
+        <div class="pageNameDiv">
+            <h1>Anani Web API</h1>
+            <h4>Client's happiness Our Matters</h4>
+        </div>
+        <div class="searchDiv">
+            <input id="search-input" type="text" class="form-control" placeholder="Search...">
+        </div>
+        <div class="tableDiv">
+            <table class="table tableWidth table-rsponsive">
+                <thead id="theadDesign">
+                    <tr>
+                        <th data-column="id" data-order="desc">ID</th>
+                        <th id="nameId"data-column="name" data-order="desc"><span class="nameSpan">Name</span></th>
+                        <th data-column="username" data-order="desc">UserName</th>
+                        <th data-column="email" data-order="desc">Email</th>
+                        <th data-column="address" data-order="desc">Address</th>
+                        <th data-column="phone" data-order="desc">Phone</th>
+                        <th data-column="website" data-order="desc">Website</th>
+                        <th data-column="company" data-order="desc">Company</th>
+                    </tr>
+                </thead>
+                <tbody id="data-output">
+                </tbody>
+            </table>
+        </div>
+        
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js" integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.min.js" integrity="sha384-7VPbUDkoPSGFnVtYi0QogXtr74QeVeeIs99Qfg5YCF+TidwNdjvaKZX19NZ/e6oz" crossorigin="anonymous"></script>
+        <script src="https://unpkg.com/bootstrap-table@1.16.0/dist/bootstrap-table.min.js"></script>  
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/5.0.0-alpha1/js/bootstrap.min.js" integrity="sha384-oesi62hOLfzrys4LxRF63OJCXdXDipiYWBnvTl9Y9/TRlw5xlKIEHpNyvvDShgf/" crossorigin="anonymous"></script>
```

Comment:
what are all these script imports ?

Reply:
as I commented before those affected the design but after we remove the script from the JS, they didn't work anymore